mpf/config_spec.yaml

```diff
@@ -954,6 +951,11 @@ mpf:
 mpf-mc:
     __valid_in__: machine                           # todo add to validator
     __type__: config
+mpfmc_segment_display_emulator:
+    __valid_in__: machine
+    __type__: config
+    bcp_connection: single|str|local_display
+    debug: single|bool|false
```

Comment:
I would expect a `source_display: single|machine(segment_displays)|` here

Reply:
This section is not used (I forgot to remove it from one of my previous attempts at a dedicated virtual segment display platform). I will fix.